### DISCUSSION TITLE
Getting Cloudflare IP instead of User's actual IP

### DISCUSSION THREAD

**Discussion by Aniket-IN** [NginxProxyManager/nginx-proxy-manager#2077] [General]
Hi,
I'm getting Cloudflare's IP instead of the user's actual IP. 
like: `162.158.170.116`, `172.70.147.188`
Previously everything was fine, but for some reason, I'm getting this issue for the past month. 

Is there way we can fix it directly from `Nginx Proxy Manager` instead of doing it in each docker container?

Thank you so much for reading.

**Comment by sdenike** [comment]
> Hi, I'm getting Cloudflare's IP instead of the user's actual IP. like: `162.158.170.116`, `172.70.147.188` Previously everything was fine, but for some reason, I'm getting this issue for the past month.
> 
> Is there way we can fix it directly from `Nginx Proxy Manager` instead of doing it in each docker container?
> 
> Thank you so much for reading.

Did you ever get this resolved? Where are you setting it per proxy host?

**Comment by trwy7** [comment]
> Is there way we can fix it directly from `Nginx Proxy Manager` instead of doing it in each docker container?

How did you do it in the container?

**Comment by CodeWithCJ** [comment]
Add this and it will start capuring REAL IP forwarded by tunnel in nginx logs

real_ip_header    X-Forwarded-For;
real_ip_recursive on;

![image](https://github.com/user-attachments/assets/b9b808a0-c14b-4e43-8f82-e7d315860350)
